RELEASE_NOTES.md

```diff
@@ -27,6 +27,17 @@
 [`int8` or `int4` compressed key and value]: https://oneapi-src.github.io/oneDNN/v3.7/dev_guide_graph_sdpa_compressed_kv.html#doxid-dev-guide-graph-sdpa-compressed-kv
 [Gated Multi-Layer Perceptron (Gated MLP)]: https://oneapi-src.github.io/oneDNN/v3.7/dev_guide_graph_gated_mlp.html#doxid-dev-guide-graph-gated-mlp
 
+## AArch64-based Processors
+  * Add support for the Arm Compute Library (ACL) thread_local scheduler via ThreadpoolScheduler
+  * Improved memory efficiency in ACL matmuls by fixing a bug where scratchpad memory was not being used
+  * The following improvements along with numerous bug fixes are carried over from the 3.6.2 patch release:
+    * Fixed a thread safety issue in matmul primitive for builds relying on Arm Compute Library (ACL) and bumped minimal supported ACL version to 24.11.1
+    * Enabled concurrent calls of ACL matmul primitive and bumped minimal supported ACL version to 24.11.1
```

Comment:
ACL version update is a breaking change.

Reply:
I believe I added it under the breaking changes as well?